src/collectors/kafkastat/test/testkafka.py

```diff
@@ -108,10 +108,10 @@ def test_query_mbean(self, get_mock):
         get_mock.return_value = self._get_xml_fixture('mbean.xml')
 
         expected_metrics = {
-            'kafka.logs.mytopic-1.CurrentOffset': long('213500615'),
-            'kafka.logs.mytopic-1.NumAppendedMessages': long('224634137'),
-            'kafka.logs.mytopic-1.NumberOfSegments': int('94'),
-            'kafka.logs.mytopic-1.Size': long('50143615339'),
+            'kafka.kafka.logs.mytopic-1.CurrentOffset': long('213500615'),
```

Comment:
I'm not really adding `kafka`. It's taken from mbean object, you can see it on line 155 (`prefix`). This is to have a separation which metrics belong to Kafka itself and which are JVM related. Like `java.lang.GarbageCollector.*` or `kafka.consumer.*`. Those will be under a default `path`, which is also `kafka` and can be changed in the Collector configuration.